quipucords/api/fingerprint/model.py

```diff
@@ -53,71 +69,98 @@ class SystemFingerprint(models.Model):
 
     system_creation_date = models.DateField(null=True)
 
-    infrastructure_type = models.CharField(
-        max_length=10, choices=INFRASTRUCTURE_TYPE)
-
-    virtualized_is_guest = models.NullBooleanField()
     virtualized_type = models.CharField(max_length=64, unique=False, null=True)
     virtualized_num_guests = models.PositiveIntegerField(
         unique=False, null=True)
     virtualized_num_running_guests = models.PositiveIntegerField(
         unique=False, null=True)
 
-    virtualized_host = models.CharField(max_length=128,
-                                        unique=False,
-                                        null=True)
-    virtualized_host_socket_count = models.PositiveIntegerField(unique=False,
-                                                                null=True)
-    virtualized_cluster = models.CharField(max_length=128,
-                                           unique=False,
-                                           null=True)
-    virtualized_datacenter = models.CharField(max_length=128,
-                                              unique=False,
-                                              null=True)
+    # VCenter scan facts
+    vm_name = models.CharField(max_length=256, unique=False, null=True)
```

Comment:
I think maybe we should merge `connection_host` and `vm_name` to `name` field.

Reply:
Done